CONTRIBUTING.md

```diff
@@ -74,9 +69,8 @@ It's usually a good idea to test locally for the following:
 ### Update docs
 
 Docs are under [site/] and use the [docsy] theme for hugo. Learn more about
-docsy from [docsy docs]. Regenerate the docs with:
-
-`make docs`
+docsy from [docsy docs]. The site itself are generated from the markdown
+automatically after commit.
```

Comment:
```
The site itself (html, css, etc.) is generated...
```
This sounds like humans only submit PRs with markdown changes, and that's it.  There's no human review of the assets?  Sounds good since looking at boilerplate deltas is largely useless.